#goal
uint8 MOUTH_EXPRESSION = 0
uint8 SYNCHRONIZED_SPEECH = 1

uint8 actionType

uint8 SMILE = 0
uint8 FROWN = 1
uint8 OPEN_MOUTH = 2
uint8 CLOSED_MOUTH = 3

uint8 mouthExpressionType
duration mouth_expression_duration

uint8 PLAY_SOUND_FILE = 0

uint8 speechType
string speech_filename

---
#state

---
#feedback

uint8 MOVING_TO_TARGET = 0
uint8 AT_TARGET = 1
uint8 feedbackState

